dev/generate_rule_metadata.py

```diff
@@ -113,10 +113,14 @@ def read_existing_default_value(rule_number, benchmark_id):
     rule_dir = os.path.join(common.rules_dir, f"{benchmark_id}/rules", f"cis_{rule_number.replace('.', '_')}")
     try:
         with open(os.path.join(rule_dir, "data.yaml"), "r") as f:
-            default_value = yml.load(f)["metadata"]["default_value"]
-            return default_value
+            data = yml.load(f)
+            default_value = data["metadata"]["default_value"]
+            if default_value is None or default_value == "":
+                print(f"{benchmark_id}/{rule_number} is missing default value - please make sure to add it manually")
+                return ""
+            return data["metadata"]["default_value"] or ""
```

Comment:
it'd be cool if we could avoid returning an empty string twice

Reply:
nice, missed it